src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/ModuleAutoArmor.kt

```diff
@@ -78,7 +77,7 @@ object ModuleAutoArmor : Module("AutoArmor", Category.COMBAT) {
                 )
             ) {
                 locked = true
-                wait(inventoryConstraints.delay.random())
+                wait { inventoryConstraints.delay.random() }
```

Comment:
This might have changed the way wait works... { ... } instead of ticks as argument.